### ISSUE TITLE
+5 bonus for CSP: require-sri-for: script style;

### ISSUE THREAD

**Issue by floatingatoll** [mozilla/http-observatory#216]
Hi, when CSP: require-sri-for: script style; is present, can we get an extra +5 points somewhere in the SRI section of scoring?

**Comment by floatingatoll** [comment]
(Shipped in Firefox 49, Chrome 54)

**Comment by fmarier** [comment]
This still hasn't shipped in Firefox (it's behind `security.csp.experimentalEnabled`) or Chrome (see https://bugs.chromium.org/p/chromium/issues/detail?id=618924#c5).

**Comment by april** [comment]
Yeah I'm happy to add support for it, but I would like to wait until somebody ships it in a browser anywhere.  :)

**Comment by wolfgang42** [comment]
@fmarier Any idea where that's documented? The [`require-sri-for`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/require-sri-for#Browser_compatibility) and [SRI](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity#Browser_compatibility) MDN pages just say it's supported, and I couldn't find anything other than your comment when I tried to figure out why FF considered it an unknown directive!

**Comment by fmarier** [comment]
Thanks @wolfgang42, I [filed a bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1422909) to get that corrected on MDN.
